examples/celery-rabbitmq/rabbitmq-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  labels:
+    name: rabbitmq-service
+  name: rabbitmq-service
+spec:
+  port: 5672
```

Comment:
There shouldn't be a port field here. You should be able to use --validate on kubectl create to validate the schemas.